spec/webview-spec.js

```diff
@@ -248,6 +248,37 @@ describe('<webview> tag', function () {
       })
     })
 
+    it('runs in the correct context', async () => {
```

Comment:
I didn't see an existing test for the unsandboxed version, but maybe there's one I've missed?

Reply:
Unsandboxed preload exposes everything by default so it is not needed.

Reply:
Technically, the test title should be "run in the correct scope". Context denotes the global JS environment, and both the page and preload run in the same context.